policy-service/pom.xml

```diff
@@ -60,6 +60,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-cache</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-redis</artifactId>
+        </dependency>
```

Comment:
Possibly need to look ad RedisTemplate, @dev-958 ?

Reply:
Yeah, will probably try and have a look at that today.